src/main/java/org/assertj/core/api/SoftProxies.java

```diff
@@ -47,30 +47,32 @@
+
+
+//  private static final Junction<MethodDescription> METHODS_NOT_TO_PROXY = methodsNamed("as").or(named("clone"));
+//                                                                                            .or(named("describedAs"))
+//                                                                                            .or(named("descriptionText"))
+//                                                                                            .or(named("getWritableAssertionInfo"))
+                                                                                            //.or(named("inBinary"))
+                                                                                            //.or(named("inHexadecimal"))
+                                                                                            //.or(named("newAbstractIterableAssert"))
+                                                                                            //.or(named("newObjectArrayAssert"))
+//                                                                                            .or(named("removeCustomAssertRelatedElementsFromStackTraceIfNeeded"))
+//                                                                                            .or(named("overridingErrorMessage"))
+//                                                                                            .or(named("usingComparator"))
+//                                                                                            .or(named("usingDefaultComparator"))
+//                                                                                            .or(named("usingElementComparator"))
+//                                                                                            .or(named("withComparatorsForElementPropertyOrFieldNames"))
+//                                                                                            .or(named("withComparatorsForElementPropertyOrFieldTypes"))
+//                                                                                            .or(named("withIterables"))
+//                                                                                            .or(named("withFailMessage"))
+//                                                                                            .or(named("withAssertionInfo"))
+//                                                                                            .or(named("withAssertionState"))
+//                                                                                            .or(named("withRepresentation"))
+//                                                                                            .or(named("withTypeComparators"))
+//                                                                                            .or(named("withThreadDumpOnError"));
 
```

Comment:
revert the commented lines, I have created https://github.com/joel-costigliola/assertj-core/issues/1378 to look into the METHODS_NOT_TO_PROXY usage. (I don't want to mix this concern with the actual PR)